detectors/azure/azurevm/README.md

```diff
@@ -0,0 +1,30 @@
+# Azure VM Resource detector
```

Comment:
Please convert this to a go doc: https://github.com/open-telemetry/opentelemetry-go/blob/main/CONTRIBUTING.md#documentation